docs/agile-development/advanced-topics/team-agreements/working-agreements.md

```diff
@@ -37,21 +37,21 @@ their own, and adjust times, communication channels, branch naming policies etc.
 
 ## Scrum Rhythm
 
-| Activity                                              | When                  | Duration | Who          | Accountable  | Goal                                                                       |
```

Comment:
nitpick: i personally prefer the expanded tables as it's easier to work with if txt mode if the entire table is formatted with spaces/tabs.

Reply:
ok, thank you for the input